tutorials/plot_artifacts_correction_ica.py

```diff
@@ -90,6 +90,15 @@
 # we can also take a look at multiple different components at once:
 ica.plot_properties(raw, picks=[1, 2], psd_args={'fmax': 35.})
 
+###############################################################################
+# Instead of opening individual figures with component properties, we can
+# also pass an instance of Raw or Epochs in ``inst`` arument to
+# ``ica.plot_components``. This would allow us to open component properties
+# interactively by clicking on individual component topomaps. In the notebook
+# this woks only when running matplotlib in interactive mode (``%matplotlib``).
+
+# uncomment the code below to test the inteactive mode of plot_components:
+# ica.plot_components(picks=range(10), inst=raw)
```

Comment:
because it makes sense only in interactive mode - if someone downloads the tutorial as notebook, without setting `%matplotlib` there would be no change. And also - the figure with components is present twice in the tutorial already.